src/auth.ts

```diff
@@ -54,6 +76,173 @@ interface GitHubAuthResponse {
 // TODO fix after https://github.com/http-party/node-portfinder/pull/115
 ((portfinder as unknown) as { basePort: number }).basePort = 9005;
 
+/**
+ * Get the global default account. Before multi-auth was implemented
+ * this was the only account.
+ */
+export function getGlobalDefaultAccount(): Account | undefined {
+  const user = configstore.get("user") as User | undefined;
+  const tokens = configstore.get("tokens") as Tokens | undefined;
+
+  // TODO: Is there ever a case where only User or Tokens is defined
+  //       and we want to accept that?
+  if (!user || !tokens) {
+    return undefined;
+  }
+
+  return {
+    user,
+    tokens,
+  };
+}
+
+/**
+ * Get the default account associated with a project directory, or the global default.
+ * @param projectDir the Firebase project directory.
+ */
+export function getProjectDefaultAccount(projectDir?: string | null): Account | undefined {
```

Comment:
Can `projectDir` really be `null`? What does that mean rather than just an empty string?

Reply:
This comes from `detectProjectRoot` which returns `string | null` ... it's a nasty type but I want the code to be honest about what `options.projectRoot` can be and both `undefined` and `null` are possible.